docs/mongocli/command/mongocli-auth-login.txt

```diff
@@ -74,7 +74,7 @@ Examples
 
 .. code-block::
 
-   To start the interactive setup:
+   To start the interactive login for your MongoDB Cloud Manager account:
```

Comment:
can you share why are we so keen on keeping Atlas out of this doc? right now this makes the docs incomplete and prone to confuse users

Reply:
do you mean we should keep this until we deprecate `mongocli atlas` commands? I'm ok with keeping atlas if that is the issue.

Reply:
`mongocli atlas` are already deprecated but they are still documented, since they are documented we need to keep this documentation correct until the commands are removed